### ISSUE TITLE
Simple NavigationDrawer example?

### ISSUE THREAD

**Issue by absolutebica** [mlaursen/react-md#302]
While I love the examples provided on your site, they all look amazing. It is very hard for someone somewhat new to React to try and implement even for simple examples. 

The NavigationDrawer is an example. I keep trying to scour the example provided code and figure out how I could insert a simple very limited sample of the NavigationDrawer. Even if all I did was insert the component and used static text and markup to pre-fill the necessary sections to make it work.  

Are you able to provide me some sample code that at the very minimum could get it to display?

**Comment by mlaursen** [comment]
Yeah, sorry about that. For most of my examples I am making the assumption you are not a beginner. I think #290 might help with that in the future though..

So a believe the _most basic_ rendering of the NavigationDrawer can be done like so:

```
import React, { Component } from 'react';
import NavigationDrawer from 'react-md/lib/NavigationDrawers';

export default class App extends Component {
  render() {
    return <NavigationDrawer />;
  }
}
```

You could try messing around with this [codepen](http://codepen.io/mlaursen03/pen/VpyEWB) as well. I have it set up with a very simple NavigationDrawer.

**Comment by absolutebica** [comment]
Perfect, thanks!
